cypress/integration/todo_mvc/delete_todo.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="cypress" />
+import faker from "faker";
+import selectors from "./selectors";
+
+const amountToDos = 5;
+
+describe("deletes items in the list", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.addToDos(amountToDos);
+  });
+
+  it("deletes a todo from the list", () => {
+    cy.log("note the ID of the first to do")
+      //no idea yet on how to do that
+      .log("delete the first to do")
+      .get(selectors.todos)
+      .first()
+      .find("button")
```

Comment:
This is very generic. If tomorrow a todo starts having 2 buttons instead of just one, this test will start failing. Is there a way you could select the delete button with something more specific such as a class?